### PR TITLE
*: add logs and minor changes.

### PR DESCRIPTION
**Pull request by disksing**
/cc @nolouch @siddontang

### CODE REVIEW

**Comment by nolouch** [comment]
LGTM

**Comment by siddontang** [comment]
CI failed

File: server/command.go [hunk 1]
```diff
@@ -33,6 +37,10 @@ func (c *conn) handleTso(req *pdpb.Request) (*pdpb.Response, error) {
 		return nil, errors.Trace(err)
 	}
 
+	if elapse := time.Since(startTime); elapse > time.Millisecond*10 {
```
Comment (siddontang) [inline server/command.go]: define a const for 10ms here.

**Comment by siddontang** [comment]
CI failed

**Pull request merged**